exercises/practice/relative-distance/RelativeDistanceTests.cs

```diff
@@ -0,0 +1,235 @@
+public class RelativeDistanceTests
+{
+    [Fact]
+    public void Direct_parent_child_relation()
+    {
+        var familyTree = new Dictionary<string, List<string>>
```

Comment:
Maybe use an array to indicate that the data is static?

```suggestion
        var familyTree = new Dictionary<string, string[]>
```